bsip-0087.md

```diff
@@ -0,0 +1,36 @@
+    BSIP: 0087
+    Title: Force Settlement Fee Ratio
+    Authors:
+      Jerry Liu bitcrab@qq.com
+    Status: Draft
+    Type: Protocol
+    Created: 2020-02-23
+    Discussion: https://github.com/bitshares/bsips/issues/260
+    Worker: TBD
+
+# Abstract
+This BSIP provide a solution to charge fee from force settlement.
+
+# Motivation
+Force settlement is an important part in smartcoin design, it provides the power to smartcoin holders to ask for collaterals with reference to feed price, and then guarantees the peg of smartcoin.
```

Comment:
Force settlement can't guarantee anything. It is merely a tool that may be used in a away that may strengthen the peg.